azure/ansible/deployment/playbooks/run-validate.yml

```diff
@@ -0,0 +1,47 @@
+---
+- name: Validate OCP Cluster
+  hosts: localhost
+
+  vars:
+    CLUSTER_NAME: z-mod-stack
+    DNS_ZONE: ibmzsw.com
+    OCP_USERNAME: ocpadmin
+
+  tasks:
+    - name: Check status of Openshift console
+      ansible.builtin.uri:
+        url: "https://console-openshift-console.apps.{{ CLUSTER_NAME }}.{{ DNS_ZONE }}/"
+        status_code: 200
+        validate_certs: false
+
+    - name: Openshift console status
+      ansible.builtin.debug:
+        msg: "Openshift console is up and running"
+
+    - name: Check login to Openshift cluster API
+      ansible.builtin.command:
+        "oc login --insecure-skip-tls-verify -u {{ OCP_USERNAME }} -p {{ OCP_PASSWORD }} --server=https://api.{{ CLUSTER_NAME }}.{{ DNS_ZONE }}:6443"
+      register: cluster_login
+      changed_when: cluster_login.rc != 0
```

Comment:
When running `oc login` on my local environment, the return code was 0 for a successful login. 

Is the logic behind this `changed_when` condition correct?

Reply:
Hope below should be fine. Please confirm.
`changed_when: cluster_login.rc == 0`